recipes/pmda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pmda" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 90ffbfa4e4ac6617e7811cd57ada57687daa654fbb176b835aec0f8b3d38f85c
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
We are now recommending people to drop the `-no-cache-dir` but that can be done in the feedstock. Merging...

Reply:
I took the [example](https://github.com/conda-forge/staged-recipes/blob/6ed6877ad238d772f49dd134d1ff19327a206d63/recipes/example/meta.yaml#L33) from the staged-repository. Should the example be changed too then?

Reply:
Yep. We are still catching up with the changes in the docs. See https://github.com/conda-forge/conda-forge.github.io/pull/653